docs/PowerStack.md

```diff
@@ -2,47 +2,123 @@
+
+## DB-15 Connectors
+
+The AMDC has 8x DB-15 connectors, one for each of the eight inverters. The DB-15 connectors are grouped into stacks of two, so there are four total stacks of DB-15 connectors across the front of the PCB. In the schematics, the top DB-15 connector in each stack is labeled as `A` and the bottom as `B`. The inverter numbering is as follows:
+
+| Silkscreen Label | Top / Bottom | Inverter # |
+|------------------|--------------|------------|
+| `CON10A`         | Top          | `INV 1`    |
+| `CON10A`         | Bottom       | `INV 2`    |
+| `CON10B`         | Top          | `INV 3`    |
+| `CON10B`         | Bottom       | `INV 4`    |
+| `CON10C`         | Top          | `INV 5`    |
+| `CON10C`         | Bottom       | `INV 6`    |
+| `CON10D`         | Top          | `INV 7`    |
+| `CON10D`         | Bottom       | `INV 8`    |
+
+
+### DB-15 Connector Pinout
+
+Each DB-15 connector uses the same pinout, but for its own inverter. For example, the top DB-15 of `CON10A` supplies the PWM and status signals for inverter #1, while the bottom DB-15 of `CON10D` is mapped to inverter #8.
+
+Each DB-15 connector has three rows of pins (uses the high-density DB-15 format). The connector has tiny labels next to each pin hole which map to the schematic labeling. The following table indicates the pinout for each connector:
+
+| Pin # | Signal Name |
+|-------|-------------|
+| 1     | `PWM_1`     |
+| 2     | `PWM_2`     |
+| 3     | `PWM_3`     |
+| 4     | `PWM_4`     |
+| 5     | `PWM_5`     |
+| 6     | `PWM_6`     |
+| 7     | `GND`       |
+| 8     | `STS_A`     |
+| 9     | `STS_B`     |
+| 10    | `STS_C`     |
+| 11    | `STS_D`     |
+| 12    | `VPS`       |
+| 13    | `VDD`       |
+| 14    | `GNDPS`     |
+| 15    | `GND`       |
+
+The mapping between `PWM*` signals in the above table and the "correct" power stack switch locations is not fixed in nature; this depends on the FPGA firmware being used. Typically, the odd-numbered PWM signals drive the high-side switches, while the even-numbered signals drive the low-side switches. However, this can be changed arbitarily in the FPGA firmware. In general, the PWM signals are just arbitrary outputs -- they can be driven to any sequence in the FPGA and used in any way in the power stack.
+
+For the default PWM firmware driver implementation, setting the duty ratio command for "PWM 0" results in inverter #1 (`INV1`): `PWM1` and `PWM2` being set to the desired duty ratios. These signals will then appear at `CON10A`, pins 1 and 2. The voltage level will toggle between 0V (AMDC `GND`) and the user-supplied `VDRIVE`. Setting "PWM 1" in firmware simply increments the `PWM*` signal locations, so would correspond to `INV1`: `PWM3` and `PWM4`. This continues on, cycles through each inverter number sequentially. The last one, "PWM 23", would appear at `INV8`: `PWM5` and `PWM6`.
```

Comment:
Are we able to *reasonably* supply some information about how the I/O in the table above maps to netlist info that is used in the Vivado as I/O for Verilog modules? (so if someone is trying to make their own custom Verilog code here, they understand what net connector 1, pin 4 will appear on). 

I am not sure that this is reasonable, given the shear number of pins involved here.

Reply:
@elsevers I have tried to add some information about this, thoughts?

https://github.com/Severson-Group/AMDC-Hardware/commit/50e269c56ce9b0220fddc92d58526b587e1f8f26

Reply:
Thanks. Let's have a quick conversation about this.

Reply:
@elsevers I've added a link to the pin mapping document: https://github.com/Severson-Group/AMDC-Hardware/commit/962ab92b0d0195542d650dd9770579d48204e877

Reply:
Updated the link to not be absolute here: https://github.com/Severson-Group/AMDC-Hardware/commit/7502eee9d42336106b127872dc1b68ff8d7b3409

Reply:
Looks good to me. I have created a PR to move this up out of the firmware section as your disclaimer:
> This section describes what is implemented in the default firmware. Note that the user can change the firmware which will invalidate the following discussion.

does not apply to this link